src/main/java/com/develhope/spring/vehicle/models/TractionControl.java

```diff
@@ -0,0 +1,26 @@
+package com.develhope.spring.vehicle.models;
+
+import com.develhope.spring.model.vehicle.optionals.Optionals;
+

```

Comment:
Iniziamo a creare ste classi in funzione di spring. Quindi inseriamo le annotazioni: @Table, @Entity